integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/OpenConditionsSpecification.groovy

```diff
@@ -13,12 +14,14 @@ import uk.gov.justice.digital.hmpps.cattool.model.TestFixture
 import uk.gov.justice.digital.hmpps.cattool.pages.CategoriserAwaitingApprovalViewPage
 import uk.gov.justice.digital.hmpps.cattool.pages.CategoriserHomePage
 import uk.gov.justice.digital.hmpps.cattool.pages.CategoriserSubmittedPage
+import uk.gov.justice.digital.hmpps.cattool.pages.CategoriserTasklistPage
 import uk.gov.justice.digital.hmpps.cattool.pages.openConditions.*
 
 import java.time.LocalDate
 
 import static uk.gov.justice.digital.hmpps.cattool.model.UserAccount.CATEGORISER_USER
 
+@org.junit.Ignore
 class OpenConditionsSpecification extends GebReportingSpec {
```

Comment:
there is a story to refactor these tests (remove and incorporate into review page etc)